app/models/solidus_avatax_certified/line.rb

```diff
@@ -29,7 +29,7 @@ def build_lines
 
     def item_line(line_item)
       {
-        number: "#{line_item.avatax_id}-LI",
+        number: "#{line_item.id}-LI",
```

Comment:
is this line item always an instance of `Spree::LineItem` ?

Reply:
I think yes because we only added `#avatax_id` to Spree::LineItemDecorator and Spree::ShipmentDecorator here https://github.com/sdtechdev/solidus_avatax_certified/commit/059da380f69236c6e661b22b2d084de17ae33579#diff-3db913c66e9c003baf2b520e52262446